packages/tasks/src/model-libraries.ts

```diff
@@ -304,6 +304,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"model.safetensors"`,
 	},
+	"mlc-llm": {
+		prettyLabel: "MLC-LLM",
+		repoName: "MLC-LLM",
+		repoUrl: "https://github.com/mlc-ai/mlc-llm",
+		docsUrl: "https://llm.mlc.ai/docs/",
+		filter: true,
```

Comment:
```suggestion
		filter: false,
```

(as per https://github.com/huggingface/huggingface.js/pull/788/files#r1670222328)